src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -39,16 +39,14 @@
     /**
      * Loads one or several .env files.
      *
-     * @param string    $path  A file to load
-     * @param ...string $paths A list of additional files to load
+     * @param ...string $paths A list of files to load
      *
      * @throws FormatException when a file has a syntax error
      * @throws PathException   when a file does not exist or is not readable
      */
-    public function load($path/*, ...$paths*/)
+    public function load(...$paths)
```

Comment:
I would keep the `$path, ...$paths` signature as it conveys the fact that at least one path is required.